scripts/generate-conda-lockfiles.sh

```diff
@@ -16,11 +16,20 @@ for TOOLCHAIN_TYPE in riscv-tools esp-tools; do
     rm -rf $LOCKFILE
 
     conda-lock \
-	    --no-mamba \
-	    --no-micromamba \
-	    -f "$REQS_DIR/chipyard.yaml" \
-	    -f "$REQS_DIR/docs.yaml" \
-	    -f "$REQS_DIR/$TOOLCHAIN_TYPE.yaml" \
-	    -p linux-64 \
-	    --lockfile $LOCKFILE
+      --no-mamba \
+      --no-micromamba \
+      -f "$REQS_DIR/chipyard.yaml" \
+      -f "$REQS_DIR/docs.yaml" \
+      -f "$REQS_DIR/$TOOLCHAIN_TYPE.yaml" \
+      -p linux-64 \
+      --lockfile $LOCKFILE
 done
+
+conda-lock \
+  --no-mamba \
+  --no-micromamba \
+  -f "$REQS_DIR/chipyard-lean.yaml" \
```

Comment:
If we do this, I'd prefer to avoid duplicating dependencies across 2x yaml files.

The lean lockfile should be generated solely from `chipyard.yaml`. The full lockfile can be generated from `chipyard.yaml + fsim_marshal.yaml`